policyengine_us/variables/gov/states/id/tax/income/credits/id_grocery_credit.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class id_grocery_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Idaho grocery credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "StateCode.ID"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.id.tax.income.credits.gc
+
+        # 100$ for each dependent
+        person1 = tax_unit.members = p.snap_threshold.Household_Size
+        person = person1("income", period) > p.snap_threshold
+        dependent = person(
+            "is_tax_unit_dependent", period
+        )  # example: [0, 0, 1, 1, 1]
+        total_dependents = tax_unit.sum(dependent)  # example sum [3]
+        dependet_amount = total_dependents * p.amount_dependent
+        # 20$ extra for each aged person
+        person_aged = person("age", period) > p.age_older_eligibility
```

Comment:
instead use:
age_head (tax_unit)
age_spouse(tax_unit)
head_aged = age_head >= 65 (parameter)
spouse_aged = age_spouse >= 65 (parameter)
- determine age head and spouse


filing_status = tax_unit("filing_status", period)

joint_amount = where(head_aged & spouse_aged, parameter two_aged, parameter one_aged)
_where(condition, output_if_true, output_if_false)_

joint = filing_status == filing_status.possible_values.JOINT

where(joint, joint_amount, other_amount)

Reply:
make a new id_grocery_credit_eligible.py with this calculation